spec/consensus/gossip/crdt.md

```diff
@@ -130,48 +129,178 @@ Consider the following definition for **Eventual Convergence**.
 
 |Eventual Convergence|
 |-----|
-| If there exists a correct process $p$ such that $e \in t_p$, then, eventually, for every correct process $q$, $e \in t_q$ or there exists a correct process $r$ such that $\bar{e} \in t_r$.|
-
-> **Note**
-> Nodes may learn of an entry deletion before learning of its addition.
+| If there exists a correct process $p \in P$ such that $e \in t_p$, then, eventually, for every correct process $q \in P$, either $e \in t_q$ or $e$ is stale in $t_q$.
```

Comment:
$t_p$ is not defined earlier. I would maybe opt for $tuples_p$, or possibly $p.tuples$ without the subscript.

the definition of _stale in_ is missing (additional comments about this below).

Reply:
thanks for the catch. it escaped from a previous write up. removed now on the follow up PR.